tests/unit_test.R

```diff
@@ -0,0 +1,38 @@
+library(kaiaulu)
+library(testthat)
+
+test_that("calling parse_gitlog with correct perceval path and an exist local path of github project", {
+  git_repo_path <- "~/Documents/GitHub/thrift/.git"
+  perceval_path <- "/home/mliu0141/.local/bin/perceval"
+
+  result <- parse_gitlog(perceval_path, git_repo_path)
```

Comment:
One problem you will run into here is on the assumption the user has the data, even if the paths follow the convention. Similar issue @leilani-reich functions will run into for these. More on this in a follow-up.